server/protocols/webadmin/webadmin-cassandra/src/test/java/org/apache/james/webadmin/routes/CassandraMigrationRoutesTest.java

```diff
@@ -145,12 +145,23 @@ public void postShouldReturnConflictWhenMigrationOnRunning() throws Exception {
     public void postShouldReturnErrorCodeWhenInvalidVersion() throws Exception {
         when(schemaVersionDAO.getCurrentSchemaVersion()).thenReturn(CompletableFuture.completedFuture(Optional.of(OLDER_VERSION)));
 
-        given()
+        Map<String, Object> errors = given()
             .body(String.valueOf("NonInt"))
         .with()
             .post("/upgrade")
         .then()
-            .statusCode(400);
+            .statusCode(400)
+            .contentType(ContentType.JSON)
+            .extract()
+            .body()
+            .jsonPath()
+            .getMap(".");
+
+        assertThat(errors)
+            .containsEntry("statusCode", 400)
+            .containsEntry("type", "InvalidArgument")
+            .containsEntry("message", "Invalid request for version upgrade")
```

Comment:
does it add anything we don't know already ?

Reply:
It just potentially explain to a user what ``400`` mean. It is redondant but armless I will let it for the moment.
@quynhn what do you do ?